### PR TITLE
Three states three actions

### PR DESCRIPTION
**Pull request by alibanihashemi**

### CODE REVIEW

**Comment by wbarfuss** [comment]
Thanks, Ali. Note that the CI checks didn't pass in your version.

File: nbs/Environments/12_EnvUncertainSocialDilemma.ipynb [hunk 1]
```diff
@@ -32,21 +32,15 @@
     "from nbdev.showdoc import *"
```
Comment (wbarfuss) [inline nbs/Environments/12_EnvUncertainSocialDilemma.ipynb]: <p>Why did you change anything here at all?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BarfussLab/pyCRLD/pull/9/discussion/#comment-1731275092'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: nbs/Environments/13_EnvTwoStatesTwoActions.ipynb [hunk 2]
```diff
@@ -0,0 +1,276 @@
+{
```
Comment (wbarfuss) [inline nbs/Environments/13_EnvTwoStatesTwoActions.ipynb]: <p>This should be called risk-reward dilemma. Two States two actions is too generic. Please make sure also to update the description to better motive the environment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BarfussLab/pyCRLD/pull/9/discussion/#comment-1731276197'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: nbs/Environments/14_EnvThreeStatesThreeActions.ipynb [hunk 3]
```diff
@@ -0,0 +1,326 @@
+{
```
Comment (wbarfuss) [inline nbs/Environments/14_EnvThreeStatesThreeActions.ipynb]: <p>This environment is too early and too research-in-progress to be include in the package. Please remove it from the pull request.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BarfussLab/pyCRLD/pull/9/discussion/#comment-1731277157'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by wbarfuss** [comment]
What is the status of this PR? I have seen you opened another one?

**Comment by alibanihashemi** [comment]
Hi Wolf,
I was trying to solve the problem with CI. Do you have any idea what might
be wrong?
Cheers,
Ali
